.Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM gitpod/workspace-full
+
+USER gitpod
+RUN sudo apt-get update -q && \
+    sudo apt-get install -yq gcc g++
+
+RUN sudo apt-get install -y --no-install-recommends \
+    autoconf \
+    ca-certificates \
+    curl \
+    environment-modules \
+    git \
+    build-essential \
+    python \
+    nano \
+    sudo \
+    unzip \
+    cmake \
+    mpich
+
+ENV LOCAL=$HOME
+ENV PROJECT=$LOCAL/hermes
+ENV INSTALL_DIR=$LOCAL/install
+ENV SPACK_DIR=$LOCAL/spack
+ENV SDS_DIR=$LOCAL/sds
+
+RUN mkdir -p $INSTALL_DIR
+
+RUN git clone https://github.com/spack/spack ${SPACK_DIR}
+RUN git clone https://xgitlab.cels.anl.gov/sds/sds-repo.git ${SDS_DIR}
+RUN git clone --branch hariharan/posix_adapter https://github.com/hariharan-devarajan/hermes ${PROJECT}
```

Comment:
If you want to add a `.Dockerfile` could you please set it to use the master branch from the main repo instead of a branch from your fork?

Reply:
Done.